airflow/utils/state.py

```diff
@@ -180,6 +184,11 @@ def color_fg(cls, state):
     A list of states indicating that a task or dag is a success state.
     """
 
+    terminated_states = frozenset([TaskInstanceState.SHUTDOWN, TaskInstanceState.RESTARTING])
```

Comment:
```suggestion
    terminating_states = frozenset([TaskInstanceState.SHUTDOWN, TaskInstanceState.RESTARTING])
```

(update docs too etc please)